### DISCUSSION TITLE
Probleme mit HmIP RF seit Version: 3.67.10.20230114

### DISCUSSION THREAD

**Discussion by Nawile** [OpenCCU/OpenCCU#2218] [🇩🇪 Generelle Diskussionen]
EDIT: Wenn man es geschrieben hat, findet man doch die Lösung. Also für alle die auch mal verzweifeln:

Es lag an dem Standort der CCU. Dort hat ein benachbartes Gerät/Kabel gestreut. Wenige CM neuer Standort und alles ist wie früher. 
Drauf gekommen bin ich über: https://homematic-forum.de/forum/viewtopic.php?f=19&t=58383&start=40#p655795

Dennoch Danke für die Arbeit und Hilfe.

Original Post:

Hallo, seitdem ich die Version: 3.67.10.20230114 installiert habe, spielt bei mir die HmIP-Schnittstelle verrückt.

Sie scheint gut zu empfangen, aber sendet höchsten sporadisch. Fehlermeldungen außer das quasi alle IP-Geräte nicht erreichbar sind gibt es nicht. Das System lief vorher Jahrelang recht stabil.

Bisherige Maßnahmen (alle klassichen Tipps, die ich finden konnte):

- Duty Cycle ist ok (läuft natürlich bei den Tests hoch, immer < 50%)
- Carrier Sens ist aktuell <= 10% (läuft beim Testen auch hoch)
- die meisten Geräte erreichen die CCU (Präsenzmelder, Tastendruck)
- alle Batterien ok getestet + getauscht
- Stromgeräte Stück für Stück vom Netz genommen
- Einzelnen Geräte auf Werkseinstellung und aus der CCU gelöscht
- Einen Dimmer (BDT) gegen neues Gerät getauscht, das dieser zuerst ausgestiegen ist, bzw. dort ist es zuerst aufgefallen
- Alle Firmwares sind aktuell
- RedMatic deaktiviert
- Beim Anlernen des neuen BDT ist mir aufgefallen, dass im Geräte-Eingang die "HmIP-CCU3" mit der HmIP-RF-Schnittstelle ebenfalls im Posteingang war. Seitdem ich diese ebenfalls einfach als fertig bestätigt habe, funktioniert quasi nichts mehr Richtung Geräte.
- Anlernen klappt nur selten und sehr hakelig (Zeitzähler läuft nicht in Echtzeit, sondern überspringt auch halbe Sekunden)
- Und RaspberryMatic habe ich ebenfalls noch einmal aufgespielt.

Hat irgendwer eine Idee oder ist ggf. in der  Version: 3.67.10.20230114?
Für Hinweise bin ich dankbar.

**Comment by MichaelN0815** [comment]
Das CCU-Gerät gibt es bei RM doch schon lange. Welche Version hattest Du denn vorher drauf. Da das Gerät aber nur Daten sichtbar macht, die vorher auch schon "da" waren, wird das keinen Einfluss auf Dein System haben. 
Eher schon der CarrierSense. Wenn Du schreibst <=10%, dann bedeutet das ja wohl, daß der Wert eher nicht bei 0-1% liegt?! 
Und alles über 5% ist verdächtig. Ab 10% sind definitiv Störungen zu erwarten. Aber es gibt da keine scharfe Grenze. Daher solltest Du mal genauer beobachten wie sich der CS über die Zeit bei Dir darstellt. Einzelne Peaks bei 10% und sonst 0% - das wäre unkritisch oder dauerhaft>5% ????

**Reply by Nawile** [reply]
@MichaelN0815, danke für dein Feedback.

Ursache war auf jeden Fall mein P2 für das Pi-hole. Den hatte ich mal vorgezogen, um dem Nachwuchs den Minicomputer zu zeigen. Da aber das Update und viele andere Dinge quasi Zeitgleich passiert sind, habe ich das nicht damit in Verbindung gebracht. 
Also als Tipp Router, CCU und Co sind räumlich besser etwas zu trennen.

Aktuell liegt der CS bei 0% und alles läuft wie früher. Nur dass ich noch einen Dimmer zum einbauen übrig habe.
Bei dem CS hatte mich gewundert hatte, dass dieser zwar hoch war, aber Meldungen von außen noch reinkamen. Aber selbst bei einem CS von unter 10% wurde Konfigurationsdaten nicht mehr versendet und Befehle nur sehr sporadisch.

Ich hatte vor der aktuellen Firmware den Vorgänger drauf, da ich versuche immer zeitnah zu aktualisieren. Daher war ich irritiert über das "neue" Gerät. Allerdings hatte ich seit dem Umstieg auf RaspberryMatic auch keine neuen Geräte mehr angelernt, da ich in letzter Zeit eher MQTT-Geräte hinzukaufe. Daher kann das auch schon eine Weile da drin gewesen sein.
